SU-HSDO/suhumsci
# ARCH-321 minor tweak to table pattern to match decanter default table styles

# READY FOR REVIEW

# Summary
- Match table pattern to Decanter default
- Uniformity for file naming conventions

# Needed By (Date)
- End of sprint (8/16)

# Urgency
- N/A

# Steps to Test

1. Pull this branch
2. Make sure table pattern matches Decanter table styles at https://su-sws.github.io/decanter/styleguide/section-elements.html#kssref-elements-tables

# Affected Projects or Products
- suhumsci (theme)
- Archaeology

# Associated Issues and/or People
- https://stanfordits.atlassian.net/browse/ARCH-321
- https://github.com/SU-SWS/decanter/pull/185

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
